eosdis-nasa/earthdata-pub-forms
# Edpub 1188 increasing forms code coverage

# Description

Adding additional unit tests to forms

## Spec

See Ticket: https://bugs.earthdata.nasa.gov/browse/EDPUB-1188

---

## Validation

1. Pull related branches locally.
2. Install dependencies
3. Run tests using npm run test:unit
4. Verify that test have succeeded
